doc/examples/registration/plot_register_rotation.py

```diff
@@ -235,3 +240,41 @@
 print()
 print(f"Expected value for scaling difference: {scale}")
 print(f"Recovered value for scaling difference: {shift_scale}")
+
+######################################################################
+# Some notes on this approach
+# =================================================================
+#
+# It should be noted that this approach relies on a couple of parameters
+# that have to be chosen ahead of time, and for which there are no clearly
+# optimal choices:
+#
+# 1. The images should have some degree of bandpass filtering
+# applied, particularly to remove high frequencies, and different choices here
+# may impact outcome. The bandpass filter also complicates matters because,
+# since the images to be registered will differ in scale and these scale
+# differences are unknown, any bandpass filter will necessarily attenuate
+# different features between the images. For example, the log-polar transformed
+# magnitude spectra don't really look "alike" in the last example here. Yet if
+# you look closely, there are some common patterns in those spectra, and they
+# do end up aligning well by phase correlation as demonstrated.
+#
+# 2. Images must be windowed using windows with cirular symmetry, to remove the
+# spectral leakage coming from image borders. There is no clearly optimal
+# choice of window.
+#
+# Finally, we note that large changes in scale will dramatically alter the
+# magnitude spectra, especially since a big change in scale will usually be
+# accompanied by some cropping and loss of information content. The literature
+# advises staying within 1.8-2x scale change. This is fine for most biological
+# imaging applications.
+#
+# Further reading:
+#
+# B.S. Reddy and B.N. Chatterji. An FFT-based technique for translation,
+# rotation and scale- invariant image registration. IEEE Trans. Image
+# Processing, 5(8):1266–1271, 1996.
+#
+# Tzimiropoulos, Georgios, and Tania Stathaki. "Robust FFT-based
+# scale-invariant image registration." In 4th SEAS DTC Technical Conference.
+# 2009.
```

Comment:
These could get DOI links — there's examples of this through the docs, something like `DOI`:tip10.34904

Reply:
Took care of those DOIs